docs/source/index.rst

```diff
@@ -52,14 +52,16 @@ Contents
 
 * :doc:`reference/index`
 * :doc:`reference/technical-overview`
-* :doc:`reference/sudo`
 * :doc:`reference/websecurity`
 * :doc:`reference/authenticators`
 * :doc:`reference/spawners`
 * :doc:`reference/services`
 * :doc:`reference/rest`
 * :doc:`reference/upgrading`
 * :doc:`reference/config-examples`
+* :doc:`reference/config-examples`
+* :doc:`reference/config-examples`
```

Comment:
reference/config-examples occurs three times here.

Reply:
good catch. airplane wifi and refresh